### DISCUSSION TITLE
questions of this project

### DISCUSSION THREAD

**Discussion by ivan-vilches** [supabase/supabase#10123] [Questions] [answered]
hello folks, this project self hosted is free ? can be used for commercial apps? self host , have limitations?
thanks you very much.

**Comment by SamMakesThings** [comment, marked as answer]
* Self hosted is free, though you'll need to bring your own server etc.
* Can definitely be used for commercial apps
* There are some limitations, e.g. no edge functions, and way more setup work. This isn't fully documented anywhere as far as I can tell.
